packages/component-model-user/entities/user/helpers/api-cache.test.js

```diff
@@ -0,0 +1,12 @@
+const ApiCache = require('./api-cache')
+
+describe('eLife API Cache tests', () => {
+  let cache
+
+  beforeEach(() => {
+    cache = new ApiCache(1)
+  })
+  it('Cache adds entries', async () => {
+    expect(cache.getLength()).toBe(0)
+  })
```

Comment:
We probably need tests for checking expiry

Reply:
yeah this file is going to end up pretty big when I finish it :)